content/installation/java/JavaOverview.md

```diff
@@ -8,3 +8,13 @@ The Contrast Java agent analyzes the behavior of Java applications running on J2
 
 ## How it Works
 To start analyzing an application, install the Java agent by adding it as `-javaagent` JVM parameter. After installation, continue to use the application as you normally would. (If it's a web application, just click on links, submit forms using normal data, and so forth.) The Java agent's sensors gather information about the application's security, architecture and libraries. You can see the results of the agent's analysis in the Contrast interface.
+
+## Note for Proguard Users
```

Comment:
I'd make this a lower-level subhead (###).

Reply:
Because it is semantically a child element of the "How it Works" heading, or just because we want smaller type?

Reply:
I was reading it as a subsection of "How It Works" - maybe because of the "Note" in the subhead. 

If that doesn't make sense from a technical/content perspective, I'm good to keep it as a level two. But, we should probably rename it something like "Proguard Optimizations" or something similar.

Reply:
I don't have strong feelings about whether or not it is a sub-heading of "How it Works" or not. It sort of feels like something miscellaneous that could belong anywhere 🤷‍♂️ - is there a better place to put it?